doc/packet_trimming/packet-trimming-design.md

```diff
@@ -1039,6 +1213,7 @@ PT basic configuration test:
 5. Verify ASIC DB object state after buffer profile discard packet action update
 6. Verify ASIC DB object state after switching `symmetric<->asymmetric` DSCP mode
 7. Verify error handling with negative test cases
+8. Verify `switch/port/queue` counter statistics
```

Comment:
can this be supported by VS test? iirc, VS test doesn't really support trimming.

Reply:
@r12f the relevant VS test cases are added to verify trimming stats. Please check `sonic-swss` implementation PR